modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -449,6 +449,7 @@ export interface MPCTx {
   recoveryAmount?: string;
   // the transaction signature used for broadcasting the transaction
   signature?: string;
+  token?: string;
```

Comment:
What's the need for a separate `token` field & why can't we just pass the token name in `coin` field?

Reply:
removed it.